creator-node/src/apiSigning.js

```diff
@@ -25,6 +25,15 @@ const generateTimestampAndSignature = (data, privateKey) => {
   return { timestamp, signature: signedResponse.signature }
 }
 
+const generateSignature = (data, privateKey) => {
+  // JSON stringify automatically removes white space given 1 param
+  const toSignStr = JSON.stringify(sortKeys(data))
+  const toSignHash = web3.utils.keccak256(toSignStr)
+  const signedResponse = web3.eth.accounts.sign(toSignHash, privateKey)
+
+  return { signature: signedResponse.signature }
+}
+
```

Comment:
Duplicate code between this and generateTimestampAndSignature - would be nice if they could use the same method